ci/cloudbuild/build.sh

```diff
@@ -242,8 +242,7 @@ if [[ "${DOCKER_FLAG}" = "true" ]]; then
   )
   cmd=(ci/cloudbuild/build.sh --local "${BUILD_NAME}")
   if [[ "${SHELL_FLAG}" = "true" ]]; then
-    io::log "Starting shell, to manually run the requested build use:"
-    echo "==> ${cmd[*]}"
+    printf "To run the build manually:\n  %s\n\n" "${cmd[*]}"
```

Comment:
`%q` perhaps?

Reply:
That would output something like `ci/cloudbuild/build.sh\ --local\ asan`, which with those escaped spaces, is not a copy-n-pasteable command. So I _think_ we want `%s`.

Context: The point of this output is that when you're trying to debug some build, and you drop to a shell, it may be useful to know the exact command that the build would have run, so that you can run it yourself and them maybe poke around at the output.

Reply:
Then I think you want:
```
printf "To run the build manually:\n"
printf " %q" "${cmd[@]}"
printf "\n\n";
```

Reply:
Thanks. Done.